contracts/committee/Committee.sol

```diff
@@ -213,159 +253,225 @@ contract Committee is ConsensusModule {
-                // insertMember will pop members beyond committee size
+                // Validator has found its correct nearer and further member
+                // insertMember will pop members beyond committee size.
                 insertMember(_validator, nearerMember, furtherMember);
                 return true;
             }
         }
 
         insertMember(_validator, SENTINEL_MEMBERS, furtherMember);
+
         return true;
     }
 
     /**
-     * Initiate cool down for committee during which objections can be raised
-     * for the members entered in the committee.
+     * @notice Initiate cool down for committee during which objections can
+     *         be raised for the members entered in the committee.
      */
     function cooldownCommittee()
         external
         onlyMember
         isOpen
     {
-        require(count == committeeSize,
-            "To close committee member count must equal committee size.");
-        assert(activationBlockHeight == 0);
+        require(
+            memberCount == committeeSize,
+            "To close committee member count must equal committee size."
+        );
+
+        assert(activationBlockHeight == uint256(0));
+
         assert(memberInitiatedCooldown == address(0));
+
         memberInitiatedCooldown = msg.sender;
+
         activationBlockHeight = block.number + COMMITTEE_FORMATION_COOLDOWN;
+
         committeeStatus = CommitteeStatus.Cooldown;
     }
 
     /**
-     * Challenge committee members during cooldown period.
-     * Must be called over Consensus to assert excluded member is a validator.
-     * Present a member that isn't present in the committee but should have been.
-     * Any excluded member invalidates the committee and member who initiated
-     * cooldown wrongly will be slashed.
+     * @notice Challenge committee members during cooldown period.
+     *
+     * @dev Must be called over Consensus to assert excluded member is
+     *      a validator. Present a member that isn't present in the
+     *      committee but should have been.
+     *      Any excluded member invalidates the committee and member who
+     *      initiated cooldown wrongly will be slashed.
      */
     function challengeCommittee(address _excludedMember)
         external
         onlyConsensus
         isCoolingDown
     {
-        require(members[_excludedMember] == address(0),
-            "Member should not already be in the committee.");
-        uint256 dBoundary = distance(shuffle(members[SENTINEL_MEMBERS]), proposal);
-        uint256 dExcludedMember = distance(shuffle(_excludedMember), proposal);
-        require(dExcludedMember < dBoundary,
-            "Member has been excluded only if distance to proposal is less than the furthest committee member.");
+        require(
+            members[_excludedMember] == address(0),
+            "Member should not already be in the committee."
+        );
+
+        uint256 dBoundary = distance(
```

Comment:
here you should use `distanceToProposal(members[SENTINEL_MEMBER])` too then

Reply:
Yes. I've grepped `distance(shuffle` ... which missed for this case as there is a new line. I will double check other places also.